### ISSUE TITLE
MathicIO.ReadWriteMonomial test fails on mips64el architecture

### ISSUE THREAD

**Issue by d-torrance** [Macaulay2/mathicgb#11]
From the [Debian build log](https://buildd.debian.org/status/fetch.php?pkg=mathicgb&arch=mips64el&ver=1.0~git20170606-1&stamp=1502639891&raw=0):

    [----------] 8 tests from MathicIO
    [ RUN      ] MathicIO.Combined
    [       OK ] MathicIO.Combined (0 ms)
    [ RUN      ] MathicIO.ReadWriteRing
    [       OK ] MathicIO.ReadWriteRing (0 ms)
    [ RUN      ] MathicIO.ReadWriteMonomial
    unknown file: Failure
    C++ exception with description "ERROR: Syntax error on line 1: Expected '<', but got no more input." thrown in the test body.
    [  FAILED  ] MathicIO.ReadWriteMonomial (1 ms)
    [ RUN      ] MathicIO.ReadWriteBasis
    [       OK ] MathicIO.ReadWriteBasis (0 ms)
    [ RUN      ] MathicIO.ReadWritePoly
    [       OK ] MathicIO.ReadWritePoly (0 ms)
    [ RUN      ] MathicIO.ReadWriteTerm
    [       OK ] MathicIO.ReadWriteTerm (0 ms)
    [ RUN      ] MathicIO.ReadWriteBaseField
    [       OK ] MathicIO.ReadWriteBaseField (0 ms)
    [ RUN      ] MathicIO.ReadWriteOrder
    [       OK ] MathicIO.ReadWriteOrder (1 ms)
    [----------] 8 tests from MathicIO (2 ms total)

**Comment by d-torrance** [comment]
This appears to be fixed by the proposed patch in #9.  It's been applied to the Debian package, and the tests are all passing on mips64el now:
https://buildd.debian.org/status/fetch.php?pkg=mathicgb&arch=mips64el&ver=1.0%7Egit20181123-2&stamp=1590948152&raw=0

```
PASS: unittest
============================================================================
Testsuite summary for mathicgb 1.0
============================================================================
# TOTAL: 1
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
```

**Comment by d-torrance** [comment]
It turns out that this wasn't actually fixed by #9 (a [second build](https://buildd.debian.org/status/fetch.php?pkg=mathicgb&arch=mips64el&ver=1.0%7Egit20170606-1&stamp=1504473217&raw=0) worked just fine).  It appears to just be a random fluke since no other Debian mips64el builds have ever failed.  Closing.
